config/build/rollup.config.js

```diff
@@ -149,12 +150,26 @@ const createConfig = (
             }),
             styles({
                 mode: "extract",
-                // We don't want to try to resolve the url() occurrences in our
-                // stylesheets. We'll leave that for consumers of the library
-                // to deal with. Otherwise we end up packaging upstream assets
-                // into our libraries when our consumers should be the ones
-                // handling asset bundling.
-                url: false,
+                minimize: true,
+                url: {
+                    publicPath: "./assets",
+                    // This resolve is for the fonts in Mathquill. We want to
+                    // bundle them into the math-input package so that
+                    // consumers of @khanacademy/math-input don't need to
+                    // _also_ depend on the mathquill package.
+                    resolve(inputUrl, baseDir) {
+                        // Some inputUrl's have a hash (eg.
+                        // font/Symbola.svg#Symbola). We use the URL object to
+                        // ensure we only have the file path and nothing else.
+                        const url = new URL(inputUrl, "http://example.com");
```

Comment:
I used the `URL` object here instead of string searching just to ensure we handle things correctly.